### PR TITLE
numa_memdev_options: add case to test two nodes with different backend

### PR DESCRIPTION
**Pull request by huangyum**
ID: 1574834
Signed-off-by: Yumei Huang <yuhuang@redhat.com>

Depends on https://github.com/avocado-framework/avocado-vt/pull/1554

### CODE REVIEW

**Comment by huangyum** [comment]
(01/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_default.prealloc: PASS (34.58 s)
 (02/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_default.no_prealloc: PASS (34.46 s)
 (03/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_bind.prealloc: PASS (34.67 s)
 (04/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_bind.no_prealloc: PASS (34.83 s)
 (05/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_interleave.prealloc: PASS (34.47 s)
 (06/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_interleave.no_prealloc: PASS (34.60 s)
 (07/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_preferred.prealloc: PASS (34.54 s)
 (08/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_preferred.no_prealloc: PASS (34.88 s)
 (09/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_default.prealloc: PASS (34.51 s)
 (10/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_default.no_prealloc: PASS (34.43 s)
 (11/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_bind.prealloc: PASS (34.72 s)
 (12/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_bind.no_prealloc: PASS (34.72 s)
 (13/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_interleave.prealloc: PASS (34.23 s)
 (14/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_interleave.no_prealloc: PASS (34.33 s)
 (15/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_preferred.prealloc: PASS (34.48 s)
 (16/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_preferred.no_prealloc: PASS (34.45 s)
 (17/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_default.prealloc: PASS (34.85 s)
 (18/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_default.no_prealloc: PASS (34.91 s)
 (19/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_bind.prealloc: PASS (34.82 s)
 (20/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_bind.no_prealloc: PASS (34.78 s)
 (21/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_interleave.prealloc: PASS (34.83 s)
 (22/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_interleave.no_prealloc: PASS (34.81 s)
 (23/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_preferred.prealloc: PASS (34.90 s)
 (24/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_preferred.no_prealloc: PASS (34.78 s)
 (25/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_default.prealloc: PASS (34.68 s)
 (26/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_default.no_prealloc: PASS (34.75 s)
 (27/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_bind.prealloc: PASS (34.79 s)
 (28/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_bind.no_prealloc: PASS (34.69 s)
 (29/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_interleave.prealloc: PASS (34.74 s)
 (30/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_interleave.no_prealloc: PASS (34.85 s)
 (31/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_preferred.prealloc: PASS (34.84 s)
 (32/32) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_preferred.no_prealloc: PASS (34.74 s)

**Comment by huangyum** [comment]
@jingzhao84 @yanan-fu Could you please help review , thanks!

**Comment by yanan-fu** [comment]
@huangyum  Please make it pass CI check first,  and the depends patch too, thanks~

**Comment by huangyum** [comment]
@yanan-fu I've checked the CI result, which got nothing to do with my patch.  I believe there is something wrong with CI check.

**Comment by jingzhao84** [comment]
(1/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_default.prealloc: PASS (30.24 s)
 (2/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_default.no_prealloc: PASS (29.77 s)
 (3/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_bind.prealloc: PASS (30.33 s)
 (4/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_bind.no_prealloc: /PASS (28.62 s)
 (5/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_interleave.prealloc: PASS (30.13 s)
 (6/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_interleave.no_prealloc: PASS (30.24 s)
 (7/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_preferred.prealloc: PASS (30.14 s)
 (8/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_preferred.no_prealloc: PASS (30.24 s)

 (1/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_default.prealloc: PASS (29.99 s)
 (2/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_default.no_prealloc: PASS (29.73 s)
 (3/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_bind.prealloc: PASS (30.32 s)
 (4/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_bind.no_prealloc: PASS (30.01 s)
 (5/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_interleave.prealloc: PASS (30.38 s)
 (6/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_interleave.no_prealloc: PASS (29.86 s)
 (7/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_preferred.prealloc: PASS (29.82 s)
 (8/8) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_preferred.no_prealloc: PASS (29.80 s)

File: qemu/tests/cfg/numa_memdev_options.cfg [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    mem_fixed = 4096
     mem_devs = "mem0 mem1"
     backend_mem = memory-backend-ram
     use_mem = no
     host-nodes_mem0 = 0
     host-nodes_mem1 = 1
-    size_mem0 = 512M
-    size_mem1 = 1024M
+    size_mem0 = 1024M
+    size_mem1 = 3072M
```
[the remark this answers is not in the document]
Comment (huangyum) [reply qemu/tests/cfg/numa_memdev_options.cfg]: I set them differently to get the memory area clearly for each memdev in later test. Thanks!

**Comment by jingzhao84** [comment]
ack-by: jinzhao@redhat.com

File: qemu/tests/cfg/numa_memdev_options.cfg [hunk 2]
```diff
@@ -39,5 +39,15 @@
         - numa_hugepage:
             backend_mem = memory-backend-file
             setup_hugepages = yes
-            mem-path = /mnt/kvm_hugepage
-            target_hugepages = 1500
+            mem-path_mem0 = /mnt/kvm_hugepage
+            mem-path_mem1 = /mnt/kvm_hugepage
```
Comment (yanan-fu) [inline qemu/tests/cfg/numa_memdev_options.cfg]: `mem-path` is ok.

File: qemu/tests/cfg/numa_memdev_options.cfg [hunk 3]
```diff
@@ -39,5 +39,15 @@
...
+            target_nodes = 0 1
+            target_num_node0 = 522
+            target_num_node1 = 1546
```
Comment (yanan-fu) [inline qemu/tests/cfg/numa_memdev_options.cfg]: This value is based on the `size_mem0`, `size_mem1`,  and for 2M hugepage.
Add a comment to explain it is necessary,  when we run 1G or other hugepage, we need modify it accordingly.

File: qemu/tests/cfg/numa_memdev_options.cfg [hunk 4]
```diff
@@ -39,5 +39,15 @@
...
+        - numa_ram_hugepage:
+            backend_mem_mem0 = memory-backend-file
+            backend_mem_mem1 = memory-backend-ram
+            setup_hugepages = yes
+            mem-path_mem0 = /mnt/kvm_hugepage
+            target_nodes = 0
+            target_num_node0 = 522
```
Comment (yanan-fu) [inline qemu/tests/cfg/numa_memdev_options.cfg]: same with above

File: qemu/tests/cfg/numa_memdev_options.cfg (hunk 3 above)
Comment (huangyum) [reply qemu/tests/cfg/numa_memdev_options.cfg]: @yanan-fu Thanks, updated.

**Comment by yanan-fu** [comment]
Acked-by: Yanan Fu <yfu@redhat.com>

**Comment by jingzhao84** [comment]
@luckyh could you help to review this pr, thanks

File: qemu/tests/cfg/numa_memdev_options.cfg [hunk 5]
```diff
@@ -40,4 +40,18 @@
             backend_mem = memory-backend-file
             setup_hugepages = yes
             mem-path = /mnt/kvm_hugepage
-            target_hugepages = 1500
+            mem-path = /mnt/kvm_hugepage
```
Comment (luckyh) [inline qemu/tests/cfg/numa_memdev_options.cfg]: Duplicated line.

**Comment by luckyh** [comment]
@MiriamDeng please help review this for ppc part, thanks.

**Comment by MiriamDeng** [comment]
@luckyh,I have to NACK this patch.Almost all the test cases failed due to
" (status: 1,    output: 'qemu-kvm: -object memory-backend-ram,policy=interleave,host-nodes=1,prealloc=no,size=3072M,id=mem-mem1: cannot bind memory to host NUMA nodes: Invalid argument')

It is because patch 1554 only considers x86's numa topolopy.As for ppc,numa topolopy is different with it.
x86
# Set hugepages on specific host numa node
# target_nodes = 0 1 2
ppc
target_nodes should be 0,8 ...

**Comment by huangyum** [comment]
@MiriamDeng It should have nothing to do with avocado-framework/avocado-vt#1554, the lines are commented there.  The problem is just that the numa topology is different on ppc and x86. I've updated this patch, pls try again. Thanks!

**Comment by MiriamDeng** [comment]
Hi hungyum,P8's and P9's numa topology are different,P8 (0,1),P9 (0,8),thanks.

**Comment by huangyum** [comment]
As discussed with @MiriamDeng , I will send an internal patch to cover P9 test.

**Comment by MiriamDeng** [comment]
I wonder if we need to update "no policy_default" in cfg file for ppc as well.

**Comment by huangyum** [comment]
@MiriamDeng Could you pls explain why?

**Comment by MiriamDeng** [comment]
There is a minor issue,is it possible for us to check if the target node has enough memory for test before binding it to guest.

**Comment by huangyum** [comment]
@MiriamDeng  I'm not sure if it is a product bug.  The ppc host I tested has two numa nodes, and one of them have 0 memory. Then qemu treats it like the host only has one node. Is it expected or something wrong?
# numactl -H
available: 2 nodes (0-1)
node 0 cpus: 0 8 16 24 32
node 0 size: 32768 MB
node 0 free: 23640 MB
node 1 cpus: 40 48 56 64 72
node 1 size: 0 MB
node 1 free: 0 MB
node distances:
node   0   1 
  0:  10  20 
  1:  20  10 
# /usr/libexec/qemu-kvm -object memory-backend-ram,policy=interleave,host-nodes=1,id=mem0,size=1G
qemu-kvm: -object memory-backend-ram,policy=interleave,host-nodes=1,id=mem0,size=1G: cannot bind memory to host NUMA nodes: Invalid argument

**Comment by MiriamDeng** [comment]
It's an extreme configuration not only on ppc host but also on x86 host in my opinions.A product bug about it has been filed,which might be fixed on rhel8.0.

**Comment by MiriamDeng** [comment]
I prefer to keep these test cases for ppc as well so they need to be tested on host with nodes attached enough memory.

**Comment by huangyum** [comment]
Fixed the python3 issue. 
(01/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_default.prealloc: PASS (28.55 s)
 (02/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_default.no_prealloc: PASS (49.37 s)
 (03/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_bind.prealloc: PASS (26.82 s)
 (04/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_bind.no_prealloc: PASS (45.00 s)
 (05/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_interleave.prealloc: PASS (49.64 s)
 (06/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_interleave.no_prealloc: PASS (26.82 s)
 (07/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.x86_64.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_preferred.prealloc: PASS (28.06 s)

**Comment by MiriamDeng** [comment]
The problem has been fixed for python3 
and all the jobs have passed
 (01/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_default.prealloc: PASS (54.30 s)
 (02/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_default.no_prealloc: PASS (55.86 s)
 (03/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_bind.prealloc: PASS (47.81 s)
 (04/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_bind.no_prealloc: PASS (43.89 s)
 (05/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_interleave.prealloc: PASS (53.87 s)
 (06/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_interleave.no_prealloc: PASS (57.40 s)
 (07/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_preferred.prealloc: PASS (47.86 s)
 (08/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_policy.policy_preferred.no_prealloc: PASS (43.66 s)
 (09/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_default.prealloc: PASS (47.05 s)
 (10/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_default.no_prealloc: PASS (43.77 s)
 (11/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_bind.prealloc: PASS (53.61 s)
 (12/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_bind.no_prealloc: PASS (43.78 s)
 (13/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_interleave.prealloc: PASS (48.76 s)
 (14/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_interleave.no_prealloc: PASS (43.68 s)
 (15/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_preferred.prealloc: PASS (43.94 s)
 (16/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_mlock.policy_preferred.no_prealloc: PASS (43.95 s)
 (17/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_default.prealloc: PASS (47.76 s)
 (18/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_default.no_prealloc: PASS (53.83 s)
 (19/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_bind.prealloc: PASS (53.66 s)
 (20/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_bind.no_prealloc: PASS (47.74 s)
 (21/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_interleave.prealloc: PASS (51.54 s)
 (22/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_interleave.no_prealloc: PASS (55.63 s)
 (23/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_preferred.prealloc: PASS (47.71 s)
 (24/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_hugepage.policy_preferred.no_prealloc: PASS (53.89 s)
 (25/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_default.prealloc: PASS (43.90 s)
 (26/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_default.no_prealloc: PASS (56.09 s)
 (27/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_bind.prealloc: PASS (53.90 s)
 (28/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_bind.no_prealloc: PASS (55.52 s)
 (29/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_interleave.prealloc: PASS (51.02 s)
 (30/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_interleave.no_prealloc: PASS (54.86 s)
 (31/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_preferred.prealloc: PASS (47.64 s)
 (32/32) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.numa_memdev_options.numa_ram_hugepage.policy_preferred.no_prealloc: PASS (51.36 s)

**Pull request merged**